nessie/src/test/java/org/apache/iceberg/nessie/TestBranchVisibility.java

```diff
@@ -22,12 +22,13 @@
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.types.Types;
 import org.junit.After;
-import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.projectnessie.error.NessieConflictException;
 import org.projectnessie.error.NessieNotFoundException;
 
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
We don't typically use static imports for methods. Is there a good reason to here?

Reply:
sorry, wasn't aware of that rule. the main reason for that was purely to have a shorter line of code and not repeat the same `Assertions.` prefix over and over again

Reply:
I think the code is nicer to read without having an `Assertions.` prefix everywhere, but I can change stuff if required. Maybe we should enforce this via a checkstyle rule then?

Reply:
Yeah, it would be great to update the checkstyle rule. I'm not sure why it isn't catching this. Maybe we don't have it applied correctly to tests.